sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/TableChange.java

```diff
@@ -628,6 +630,16 @@ public int hashCode() {
       result = 31 * result + Arrays.hashCode(fieldNames);
       return result;
     }
+
+    @Override
+    public String toString() {
```

Comment:
can we do it in a separated PR? all `ColumnChange` implementations need this change.

Reply:
https://github.com/apache/spark/pull/40583/files#diff-8d2ca0d05dd54468ef4faf6ae318ed2288c46be701f3858eecbb371c8f0d420bR343 OK, I will break the test part to line-to-line comparison.